eng/Versions.props

```diff
@@ -90,14 +90,14 @@
     <MicrosoftExtensionsDiagnosticsHealthChecksLTSVersion>8.0.12</MicrosoftExtensionsDiagnosticsHealthChecksLTSVersion>
     <MicrosoftExtensionsFeaturesLTSVersion>8.0.12</MicrosoftExtensionsFeaturesLTSVersion>
     <!-- Runtime -->
-    <MicrosoftExtensionsHostingAbstractionsLTSVersion>8.0.1</MicrosoftExtensionsHostingAbstractionsLTSVersion>
+    <MicrosoftExtensionsHostingAbstractionsLTSVersion>9.0.0</MicrosoftExtensionsHostingAbstractionsLTSVersion>
```

Comment:
These are the LTS versions, so they should not be bumped to 9.0. There is the `<MicrosoftExtensionsHostingAbstractions>` tag that tracks it.

Reply:
ah - didn't think of support angle. @eerhardt I think you were already looking at this, perhaps I should clsoe this PR